spec/amp-html-format.md

```diff
@@ -192,8 +192,8 @@ Pseudo selectors are generally forbidden and must be whitelisted 1 by 1. Initial
 
 CSS variables are used to style custom elements.
 
-Class names in author stylesheets may not starts with the string `-amp-`. These are reserved for internal use by the AMP runtime. It follows,
-that user stylesheet may not reference CSS selectors for `-amp-` classes and `i-amp` tags. These classes and elements are not meant to
```

Comment:
Well, technically, there's only one user stylesheet.  How about "the user's stylesheet"